openapi/components/schemas/GatewayAccountConfig/PayRedeem.yaml

```
@@ -0,0 +1,18 @@
+description: PayRedeem config
```

nitpick: for readability, we should avoid abbreviation
```suggestion
description: PayRedeem gateway configuration.
```